src/modules/dashboard/sagas/utils/enableExtension.ts

```diff
@@ -0,0 +1,76 @@
+import { fork } from 'redux-saga/effects';
```

Comment:
The file name was a bit confusing to me. I would suggest something like `extensionEnableFunctions` or `extensionEnableHelpers`, but it's a nitpick - up to you.

Reply:
I agree with you! It could have a better name